packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -401,10 +401,14 @@ describe("Data Store Context Tests", () => {
 				const expectedEvents = [
 					{
 						eventName: "FluidDataStoreContext:DataStoreCreatedInSummarizer",
-						packageName: packagePathToTelemetryProperty(packageName),
+						fullPackageName: {
```

Comment:
Why not use `tagCodeArtifacts`?

Reply:
i prefer test data to be explicit, if the code to generate the data is wrong, and the test use it, it may be missed